substrate-processor/src/processor/handlerProcessor.ts

```diff
@@ -630,6 +635,58 @@ export class SubstrateProcessor<Store> {
         return this
     }
 
+    /**
+     * Registers `EVM.Executed` event handler.
+     *
+     * This method is similar to {@link .addEventHandler},
+     * but provides specialised {@link EvmExecutedEvent | event type}.
+     * 
+     * `EVM.Executed` event contains all the logs emitted during a contract call
+     * and this handler selects events by evm log contract address and topics
+     * contained in them.
+     *
+     * @example
+     * // process ERC20 transfers from Karura contract
+     * processor.addEvmExecutedHandler('0x0000000000000000000100000000000000000084', {
+     *     topics: ['0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef']
+     * }, async ctx => {})
+     */
+    addEvmExecutedHandler(
```

Comment:
I suggest to name it `addAcalaEvmExecutedHandler`, because `EVM.Executed` is ambiguous and present both in Acala and Frontier pallets. 

@dzhelezov

Reply:
I don't mind. I wanted to prepare this pr and discuss naming